### DISCUSSION TITLE
hello,I compiled OBS version 30.12 using VS2022, but after the compilation, there were no include and lib files included in the installation path.How can I obtain the lib and include files?

### DISCUSSION THREAD

**Discussion by YuuKiLeeNa** [obsproject/obs-studio#10505] [Development Questions] [answered]
![K8FHYGD`~`(RBYNU${U(W84](https://github.com/obsproject/obs-studio/assets/39462742/c3306c1a-81b0-4f64-93b2-049affbb406e)

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
